packages/types/script/get-proto.sh

```diff
@@ -35,12 +35,12 @@ mkdir $PROTO_DIR
 
 # Mv the proto file into the proto dir
 mv "$TMP_DIR/desmos-$DESMOS_VERSION/proto" $PROTO_DIR
-mv "$TMP_DIR/desmos-$DESMOS_VERSION/third_party" $PROTO_DIR
 
-# Remove all the old proto folders
-rm -r "$PROTO_DIR/proto/desmos/profiles/v1beta1"
-rm -r "$PROTO_DIR/proto/desmos/profiles/v2"
-rm -r "$PROTO_DIR/proto/desmos/subspaces/v1"
+# Install needed third party proto files by telescope
+yarn telescope install @protobufs/cosmos @protobufs/cosmos_proto @protobufs/ibc
```

Comment:
The third party folder inside `desmos` is removed, we should download needed proto files manually. Thanks to `telescope`, we can install the needed proto files by a command.